middle_end/flambda2/from_lambda/closure_conversion_aux.ml

```diff
@@ -632,6 +639,17 @@ module Acc = struct
     let cost_metrics = cost_metrics acc in
     cost_metrics, free_names, with_cost_metrics saved_cost_metrics acc, return
 
+  let add_code_offsets t code_id =
```

Comment:
I think I would prefer `add_offsets_from_code`. Here I get the impression that it registers the offsets and associate them to the given code ID (which is done in `pop_closure_info` and not exported)

Reply:
renamed